README.md

```diff
@@ -119,9 +119,7 @@ saveCityState(cityStateRegex.match(cityStateRegex)[1], cityStateRegex.match(city
 **Good**:
 ```javascript
 const cityStateRegex = /^(.+)[,\\s]+(.+?)\s*(\d{5})?$/;
-const match = cityStateRegex.match(cityStateRegex)
-const city = match[1];
-const state = match[2];
+const [, city, state] = cityStateRegex.match(cityStateRegex);
```

Comment:
Yes, i will also add a string for the example because apply a regex on a regex is a non sense :-)